Userland/Libraries/LibJS/Tests/try-catch-finally-nested.js

```diff
@@ -53,3 +53,47 @@ test("Nested try/catch/finally with exceptions", () => {
     expect(level3CatchHasBeenExecuted).toBeTrue();
     expect(level3FinallyHasBeenExecuted).toBeTrue();
 });
+
+test("Nested try/catch/finally with return in inner context", () => {
+    success = false;
+    try {
+        try {
+            return;
```

Comment:
:thinking: Doesn't this return mean that while we may set success to true, we don't actually run the expect on line 68? The try expressions for these tests should be in an IIFE probably. does this problem exist for other try + return tests?